xfel/conda_envs/psana_lcls_environment.yml

```diff
@@ -0,0 +1,72 @@
+name: psana_env
+channels:
+ - file:///reg/g/psdm/sw/conda1/channels/system-rhel7
+ - lcls-i
+ - conda-forge
+ - cctbx
+ - defaults
+dependencies:
+ # conda compilers
+ - c-compiler
+ - cxx-compiler
+
+ # mpi
+ - mpi4py
+ - openmpi
+
+ # psana
+ - psana-conda  >= 2.0.9
+
+ # python
+ - biopython
+ - ipython
+
+ # testing
+ - pytest
+ - pytest-xdist
+ - pytest-mock
+
+ # documentation
+ - docutils
+ - numpydoc
+ - sphinx
+
+ # graphics
+ - pillow
+ - reportlab
+ - wxpython
+ - pyopengl
+
+ # HDF5 (main libraries come with psana)
+ - hdf5_plugins                      # [unix]
+
+ # other
+ - libsvm
+ - mrcfile
+ - psutil
+ - distro
+
+ # cctbx channel
+ - libsvm_py
+
+ # dials
+ - jinja2
+ - msgpack-python
+ - orderedset
+ - procrunner
+ - python-blosc
+ - scikit-learn
+ - tqdm
+ - eigen
+ - dials-data
+
+ # xia2
+ - tabulate
+
+ # Phenix
+ - send2trash
+ - PyRTF
```

Comment:
The Phenix GUI uses these, so they're probably not needed.

Reply:
re this comment and the previous one: sphinx, pyrtf, and docutils are in the dials environments (https://github.com/dials/dials/blob/master/.conda-envs/linux.txt et al). My preference (as well as Aaron's suggestion) is to do the same here, to prevent any surprises, maybe at the cost of carrying around some unnecessary stuff. I'll remove send2trash and numpydoc. Sound ok?